multiversx_sdk_cli/contracts.py

```diff
@@ -202,7 +201,7 @@ def query_contract(self,
                        contract_address: IAddress,
                        proxy: INetworkProvider,
                        function: str,
-                       arguments: List[Any],
+                       arguments: Union[List[Any], None],
```

Comment:
Can also be `Optional` - or maybe not yet (older Python)?

Reply:
changed to `Optional[List[Any]]`.